pipeline/lib/pipeline.ml

```diff
@@ -37,7 +37,12 @@ let read_channel_uri p =
   Util.read_fpath p |> String.trim |> Uri.of_string |> Current_slack.channel
 
 (* Generate a Dockerfile for building all the opam packages in the build context. *)
-let dockerfile ~base =
+let dockerfile ~base ~repository =
+  let opam_dependencies =
+    if String.compare repository "dune" == 0 then
```

Comment:
I think this deserves a comment – either a `HACK` or `FIXME` to make it clear that this is not ok long-term.

Reply:
Also, why not `String.equal`?

Reply:
I feel like having this as format is strange.

Maybe let's use a string here and move the conversion down as:

```ocaml
  @@ run "%s" opam_dependencies
```